### ISSUE TITLE
about 5 fold subject-exclusive cross-validation

### ISSUE THREAD

**Issue by wangyitao1234** [AnweshCR7/RhythmNet#20]
Hi，
Combining the description of 5-fold subject-exclusive cross-validation in the paper and your description in README.MD, I would like to ask if my understanding of this is correct? We classify 2-fold, 3-fold, 4-fold, and 5-flod as training sets, conduct five-fold cross-validation on the training set, then select the model with the highest average accuracy, and finally use this model to test 1-flod  and the results you get are the results in your readme.md.
Looking forward to your reply
